src/Video/Texture/Texture2D.cpp

```diff
@@ -31,12 +31,12 @@ GLuint Texture2D::GetTextureID() const {
     return texID;
 }
 
-int Texture2D::GetWidth() const {		
-    return width;		
-}		
- 		
-int Texture2D::GetHeight() const {		
-    return height;		
+int Texture2D::GetWidth() const {
+    return width;
+}
+        
+int Texture2D::GetHeight() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetHeight' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)